spec/rails_helper.rb

```diff
@@ -61,4 +61,12 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+end
+
+Shoulda::Matchers.configure do |config|
```

Comment:
Glad we figured out the error was just a gem issue and not a break in our database!

Reply:
Seriously glad we didn't break into our database!  Data no Databasing! (Swiper no Swiping!)